R/geom-defaults.r

```diff
@@ -7,14 +7,42 @@
 #' @keywords internal
 #' @export
 #' @examples
-#' update_geom_defaults("point", list(colour = "darkblue"))
+#'
+#' # updating a geom's default aesthetic settings
+#' # example: change geom_point()'s default color
+#' GeomPoint$default_aes
+#' update_geom_defaults("point", list(color = "red"))
+#' GeomPoint$default_aes
 #' ggplot(mtcars, aes(mpg, wt)) + geom_point()
+#'
+#' # reset default
 #' update_geom_defaults("point", list(colour = "black"))
+#'
+#' # note: adding default aesthetics is possible
+#' # update_geom_defaults("point", list(foo = "bar"))
+#' # GeomPoint$default_aes
+#'
+#'
```

Comment:
I think it is fine to omit this, this usage isn't advertised as it is easy to get yourself in trouble if you don't know what you're doing and end up breaking a geom.